### ISSUE TITLE
Integrations

### ISSUE THREAD

**Issue by adclaran** [alicolville/Weight-Tracker#469]
Hi
I'm using GamiPress and WPForms. Both are excellent plugins with a lot of users. You could consider integration with that too, beside mycred and gravity. Thanks!

**Comment by YOPT21** [comment]
WPforms integration would be good to have

**Comment by alicolville** [comment]
Worth saying there is a myCred integration: https://docs.yeken.uk/gamification.html

**Comment by alicolville** [comment]
Also one for GravityForms https://docs.yeken.uk/gravity-forms.html

**Comment by YOPT21** [comment]
Any joys with a wordpress form or elementor form intergration?

**Comment by alicolville** [comment]
Afraid not.

**Comment by YOPT21** [comment]
Opening this integration would be beneficial but not necessarily as important as other updates you may have.

**Comment by adclaran** [comment]
Yes, that would be awesome! WPForms have the most updated program... and tons of features... GamiPress would have great benefit of an integration.

**Comment by alicolville** [comment]
Is this still something to consider doing?
